make_release/Readme.md

```diff
@@ -1,8 +1,22 @@
 # The release process of Nushell
 ## 0. Check dependencies
-- [ ] [`reedline`] is updated and released ([example][reedline bump example])
-- [ ] pin [`reedline`] in [`nushell`] ([example][reedline pin example])
-- [ ] [`nu-ansi-term`] stays the same
+> **Note**  
+> the following procedure is the same for `nu-ansi-term` and `reedline` and needs to be repeated
+
+> **Warning**  
+> release `nu-ansi-term` **before** `reedline` and `reedline` **before** Nushell
+
+> **Note**  
+> `nu-ansi-term` is typically released only when there are changes to publish.
+> `reedline` is typically released on the same schedule as Nushell.
+
+- [ ] bump the version (example with [`reedline`][reedline bump example] and [`nu-ansi-term`][nu-ansi-term bump example])
+- [ ] get the latest revision with `git pull origin main`
```

Comment:
Let's skip specifying the remote here. As `origin` is often the personal fork and `upstream` may be convention but not guaranteed.

Reply:
it've tried being more general and less ambiguous in d67bbf9 :innocent: